packages/bisect_ppx/bisect_ppx.0.2.2/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+name: "bisect_ppx"
+version: "0.2.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/bisect_ppx"
+dev-repo: "https://github.com/rleonid/bisect_ppx.git"
+bug-reports: "https://github.com/rleonid/bisect_ppx/issues"
+build: [
+  ["sh" "configure"]
+  [make "all"]
+]
+install: [
+  [make "install"]
+  ["cp" "bisect-report" bin]
+  ["cp" "bisect-report.opt" bin]
```

Comment:
I think you shouldn't install both of these. The .opt should be installed under the same name if ocamlopt is available.